### ISSUE TITLE
qwopqwop200/GPTQ-for-LLaMa breakage

### ISSUE THREAD

**Issue by alexl83** [oobabooga/text-generation-webui#1343]
### Describe the bug

Recent `import` statements removal prevent textgen from starting in 4bit mode
likely qwopqwop200/GPTQ-for-LLaMa culprit commits:
`2e673b8f3ae2b39f5115e305b63cc02b07bf6c20` and `3569be69c2b118c975a955e05ef28df9dcea3aff`

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

pull latest 'triton' and try starting textgen with a 4bit model

### Logs

```shell
Loading settings from /home/alex/oobabooga/settings.json...
Loading llama-30b-oa-4bit-128g...
Traceback (most recent call last):
  File "/home/alex/oobabooga/text-generation-webui/server.py", line 916, in <module>
    shared.model, shared.tokenizer = load_model(shared.model_name)
  File "/home/alex/oobabooga/text-generation-webui/modules/models.py", line 125, in load_model
    from modules.GPTQ_loader import load_quantized
  File "/home/alex/oobabooga/text-generation-webui/modules/GPTQ_loader.py", line 14, in <module>
    import llama_inference_offload
  File "/home/alex/oobabooga/text-generation-webui/repositories/GPTQ-for-LLaMa/llama_inference_offload.py", line 8, in <module>
    from utils import find_layers, DEV, set_seed, get_wikitext2, get_ptb, get_c4, get_ptb_new, get_c4_new, get_loaders
  File "/home/alex/oobabooga/text-generation-webui/repositories/GPTQ-for-LLaMa/utils/__init__.py", line 1, in <module>
    from modelutils import DEV, find_layers
ModuleNotFoundError: No module named 'modelutils'
ERROR conda.cli.main_run:execute(47): `conda run python server.py --wbits 4 --chat --listen --settings /home/alex/oobabooga/settings.json --model llama-30b-oa-4bit-128g` failed. (See above for error)
```

`pip install modelutils` doesn't help:

```
Loading settings from /home/alex/oobabooga/settings.json...
Loading llama-30b-oa-4bit-128g...
Traceback (most recent call last):
  File "/home/alex/oobabooga/text-generation-webui/server.py", line 916, in <module>
    shared.model, shared.tokenizer = load_model(shared.model_name)
  File "/home/alex/oobabooga/text-generation-webui/modules/models.py", line 125, in load_model
    from modules.GPTQ_loader import load_quantized
  File "/home/alex/oobabooga/text-generation-webui/modules/GPTQ_loader.py", line 14, in <module>
    import llama_inference_offload
  File "/home/alex/oobabooga/text-generation-webui/repositories/GPTQ-for-LLaMa/llama_inference_offload.py", line 8, in <module>
    from utils import find_layers, DEV, set_seed, get_wikitext2, get_ptb, get_c4, get_ptb_new, get_c4_new, get_loaders
  File "/home/alex/oobabooga/text-generation-webui/repositories/GPTQ-for-LLaMa/utils/__init__.py", line 1, in <module>
    from modelutils import DEV, find_layers
ModuleNotFoundError: No module named 'modelutils'
ERROR conda.cli.main_run:execute(47): `conda run python server.py --wbits 4 --chat --listen --settings /home/alex/oobabooga/settings.json --model llama-30b-oa-4bit-128g` failed. (See above for error)
```
```


### System Info

```shell
Ubuntu 22.04 amd64
RTX4090
Ryzen 7700X
micromamba env
```

**Comment by Ph0rk0z** [comment]
change to `from utils import find_layers`

GPTQ just *love* breaking changes.

**Comment by xNul** [comment]
Only [this fork](https://github.com/oobabooga/GPTQ-for-LLaMa) of GPTQ-for-LLaMA is currently supported.

**Comment by Thireus** [comment]
> change to `from utils import find_layers`
> 
> GPTQ just _love_ breaking changes.

I can confirm that fixes it.

**Comment by alexl83** [comment]
Fixed by e1aa9d5173368015fd7e21d582d4fe620adee901
